sqlstore/src/main/resources/db/migration_afterMigrate/05_standard.sql

```diff
@@ -940,6 +940,9 @@ AS SELECT
     d.concentration AS dilutionConcentration,
     d.identificationBarcode AS dilutionBarcode,
     d.lastUpdated AS lastModified,
+    d.creationDate AS created,
+    d.dilutionUserName AS creatorName,
+    (SELECT loginName FROM User WHERE userId = d.lastModifier) AS lastModifierName,
```

Comment:
Looks like we should prefer a join. Ready to run pools request was taking 7.5 seconds with the subquery, but is down to 1.5 seconds when I changed it to a join. Will include the change in GLT-1682